### PR TITLE
Add basic log loading class

### PR DESCRIPTION
**Pull request by breadcentric**

### CODE REVIEW

File: deepracer/logs/log.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        self.model_folder = model_folder
+        
+        self.iter_count = {}
+
+        self.df = None
+
+    def load_model_iteration(self, model_iteration):
+        dfs = []
+
+        dfs = [self.load_worker(model_iteration, worker_id) for worker_id in worker_ids]
```
Comment (breadcentric) [inline deepracer/logs/log.py]: load_worker and worker_ids are not declared

Comment (breadcentric) [reply deepracer/logs/log.py]: Do we need this method at all? Looks like load is doing it all

File: deepracer/logs/log.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+    def dataframe(self):
+        """Method that provides the dataframe for analysis of this log.
+
+        Raises:
+            NotImplementedError: This class should not be used.
+        """
+        raise NotImplementedError("Implement this in a subclass!")
+
+    def hyperparameters(self):
```
Comment (jochem725) [inline deepracer/logs/log.py]: `action_space` and `agent_and_network` should also be defined here

**Review by jochem725 — CHANGES_REQUESTED** [review]

File: deepracer/logs/log.py (hunk 2 above)
Comment (breadcentric) [reply deepracer/logs/log.py]: Done

**Review by breadcentric — COMMENTED** [review]
I think we're good to release now.

**Comment by breadcentric** [comment]
I have discussed this with Jochem in the meantime and believe this is ready to release.

**Pull request merged**